lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
Lets keep this simple: cache the tag and always return the same command and rows.